processing/src/main/java/io/druid/query/lookup/LookupReferencesManager.java

```diff
@@ -233,9 +318,20 @@ public boolean remove(String lookupName)
   @Nullable
   public LookupExtractorFactory get(String lookupName)
   {
-    final LookupExtractorFactory lookupExtractorFactory = lookupMap.get(lookupName);
-    assertStarted();
-    return lookupExtractorFactory;
+    try {
```

Comment:
It is a read lock, so it should be fast unless there's a start/stop state change going on, in which case it should fail.